mailpoet/tasks/phpstan/email-editor-phpstan-custom.php

```
@@ -0,0 +1,12 @@
+<?php declare(strict_types = 1);
+
+$config = [];
+$phpVersion = (int)getenv('ANALYSIS_PHP_VERSION') ?: PHP_VERSION_ID;
+$config['parameters']['phpVersion'] = $phpVersion;
+
+// Load PHP 8 specific config
+if ($phpVersion >= 80000) {
```

I guess this won't work on PHP 8.1 because the ignored error in the PHP 8 config is related only to PHP 8.2 and higher. It may not be a big deal, but we should be accurate with those conditions to avoid confusion among other developers because they can use this version.

---

Fixed in https://github.com/mailpoet/mailpoet/pull/5943/commits/68d976b1fb919949ef1ee044c25b6660f610e5c1